tests/integration/test_ethereum_tester.py

```diff
@@ -175,18 +175,56 @@ def inner(*args, **kwargs):
     return inner
 
 
-class TestEthereumTesterEthModule(EthModuleTest):
-    test_eth_sign = not_implemented(EthModuleTest.test_eth_sign, ValueError)
-
-    def test_eth_getTransactionReceipt_unmined(self, eth_tester, web3, unlocked_account):
+def disable_auto_mine(func):
+    @functools.wraps(func)
+    def func_wrapper(self, eth_tester, web3, unlocked_account):
```

Comment:
Since you're not doing anything with the other args, this could be `(self, eth_tester, *args)`, for a little more generality if the wrapper gets used on a method with a different signature.